README.md

```diff
@@ -55,7 +55,7 @@ The main concept is to create a new instance of the `SpvWalletClient` class and
 ```typescript
 import { SpvWalletClient } from '@bsv/spv-wallet-js-client';
 
-const spvWalletServerUrl = 'http://localhost:3003/v1';
+const spvWalletServerUrl = 'http://localhost:3003/api/api/v1';
```

Comment:
I think it should be: `http://localhost:3003/api/v1`

Reply:
fixed